config/initializers/hesa_degree_types.rb

```diff
@@ -0,0 +1,85 @@
+HESA_DEGREE_TYPES = [
```

Comment:
Would it be better to use a hash for each entry in the array so that the keys would provide a little more context on what each value is/means?

Reply:
This was sourced directly from 

https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/config/initializers/hesa_degree_types.rb

There are a degree spike, to look at these in greater details

Other then the 3 field everything else is rather useless at present